lib/global-styles.php

```diff
@@ -521,11 +521,16 @@ function gutenberg_experimental_global_styles_merge_trees( $core, $theme, $user
 	$result = gutenberg_experimental_global_styles_normalize_schema( array() );
 
 	foreach ( array_keys( $core ) as $block_name ) {
-		foreach ( array( 'presets', 'features' ) as $subtree ) {
-			$result[ $block_name ][ $subtree ] = array_merge(
-				$core[ $block_name ][ $subtree ],
-				$theme[ $block_name ][ $subtree ],
-				$user[ $block_name ][ $subtree ]
+		$result[ $block_name ][ 'presets' ] = array_merge(
+			$core[ $block_name ][ 'presets' ],
+			$theme[ $block_name ][ 'presets' ],
+			$user[ $block_name ][ 'presets' ]
+		);
+		foreach ( array_keys( $core[ $block_name ]['features'] ) as $subtree ) {
```

Comment:
The thing is here, I'm not sure we will only have arrays here, maybe some keys in "features" will contain scalar values in the end.

Reply:
Ah, right. This won't work _if_ we do that.

Given that we don't have that use case yet, that we're coming with a big refactoring of theme.json, and that the `colors` is now broken, would you think we can land this PR as it is and iterate?

Reply:
Yes, I'm ok with that.